packages/core/src/middleware/koa-connector-error-handler.test.ts

```diff
@@ -0,0 +1,169 @@
+import { ConnectorError, ConnectorErrorCodes } from '@logto/connector-types';
+
+import RequestError from '@/errors/RequestError';
+import { createContextWithRouteParameters } from '@/utils/test-utils';
+
+import koaConnectorErrorHandler from './koa-connector-error-handler';
+
+describe('koaConnectorErrorHandler middleware', () => {
+  const next = jest.fn();
+  const ctx = createContextWithRouteParameters();
+
+  it('should throw no errors if no errors are catched', async () => {
+    await expect(koaConnectorErrorHandler()(ctx, next)).resolves.not.toThrow();
+  });
+
+  it('should throw original error if error type is no ConnectorError', async () => {
```

Comment:
is `not` ConnectorError?

Reply:
This test case is tricky, because technically ConnectorError will also be thrown as its original error form, which is ConnectorError, right?